src/format.ts

```diff
@@ -32,12 +32,10 @@ export async function format(
   const program = createProgram(options);
   const srcFiles = files.length > 0 ?
       files :
-      program.getSourceFiles()
-          .map(sourceFile => sourceFile.fileName)
-          .filter(f => !f.endsWith('.d.ts'));
+      program.getRootFileNames().filter(f => !f.endsWith('.d.ts'));
```

Comment:
Can you briefly comment here on why only root files should be formatted?

Reply:
Done